src/Module/BaseApi.php

```diff
@@ -195,80 +187,108 @@ public static function getRequest(array $defaults)
 	 *               'authenticated' => return status,
 	 *               'user_record' => return authenticated user record
 	 */
-	protected static function login(string $scope)
+	protected static function login(string $scope): bool
```

Comment:
The return value of this function is not needed at all. It is designed to stop the execution when not logged in. So either it stops or the login worked. There is no need checking the return value at all.

Reply:
This was just a suggestion from my IDE, don't read too much into it. If this method isn't meant to have a return value, it shouldn't have a return statement and probably should be named something different to be clearer.

Reply:
I've now cleaned up this part.

Reply:
It had a return value from the start. I hadn't removed it yet, but had it in my mind, since at some places in the code the return value is used.

Reply:
Now the confusion is cleared.